mitogen/__init__.py

```diff
@@ -35,7 +35,7 @@
 
 
 #: Library version as a tuple.
-__version__ = (0, 3, 0)
+__version__ = (0, 3, 1, 'dev0')
```

Comment:
Just noting this is still `dev0`, needs change before release

Reply:
thanks, plan is to change that together with setting a release date in the changelog.